lib/trento/sap_systems/commands/deregister_sap_system.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Trento.SapSystems.Commands.DeregisterSapSystem do
+  @moduledoc """
+  Deregister (decommission) a SapSystem instance from the monitoring system.
```

Comment:
Remove the `instance` word.
Actually, i would simply write `Deregister (decommission) a SAP system

Reply:
done